s2/region_term_indexer_test.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2025 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s2
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+type QueryType int
+
+const (
+	POINT QueryType = iota + 1
+	CAP
+)
+
+var NUM_ITERATIONS = int(100)
+
+func testRandomCaps(t *testing.T, option Options, queryType QueryType) {
```

Comment:
Could the tests which call this be structured as a [table test](https://go.dev/wiki/TableDrivenTests)?

If not, should this function call t.Helper?